api/v1alpha1/clienttrafficpolicy_types.go

```diff
@@ -104,6 +104,12 @@ type HeaderSettings struct {
 	// +optional
 	EnableEnvoyHeaders *bool `json:"enableEnvoyHeaders,omitempty"`
 
+	// DisableRateLimitHeaders configures Envoy Proxy to omit the "X-RateLimit" headers.
```

Comment:
```suggestion
	// DisableRateLimitHeaders configures Envoy Proxy to omit the "X-RateLimit-" headers
	when rate limiting is configured.
```

Reply:
"when rate limiting is enabled" to avoid "configures ... configured"